shardingsphere-mode/shardingsphere-mode-type/pom.xml

```diff
@@ -31,5 +31,6 @@
     <modules>
         <module>shardingsphere-standalone-mode</module>
         <module>shardingsphere-cluster-mode</module>
+        <!--        <module>shardingsphere-cluster-mode-repository-consul</module>-->
```

Comment:
What is the useful

Reply:
not used, this consul repository add error module at before commit,now is deleted